backend/alembic/versions/67f5b6b8aa36_merge_moderation.py

```diff
@@ -0,0 +1,24 @@
+"""merge_moderation
+
+Revision ID: 67f5b6b8aa36
+Revises: 3fba41121353, 878310d42e29
+Create Date: 2023-04-27 18:52:14.274197
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '67f5b6b8aa36'
+down_revision = ('3fba41121353', '878310d42e29')
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
This migration does nothing?

Reply:
It's a merge migration, because another migration was added while I was working on this PR. I'll remove it when I make the other changes to the migration.